src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/ProviderRelationship/ProviderRelationshipsService.cs

```diff
@@ -61,6 +60,28 @@ public async Task<IEnumerable<EmployerInfo>> GetLegalEntitiesForProviderAsync(lo
             return await GetEmployerInfosAsync(providerPermissions);
         }
 
+        public async Task RevokeProviderPermissionToRecruitAsync(long ukprn, string accountLegalEntityPublicHashedId)
+        {
+            var httpClient = CreateHttpClient(_configuration);
+            var stringContent = GetStringContent(ukprn, accountLegalEntityPublicHashedId);
+            var uri = new Uri(new Uri(_configuration.ApiBaseUrl), "permissions/revoke");
+
+            var response = await httpClient.PostAsync(uri, stringContent);
+
+            if(!response.IsSuccessStatusCode)
+            {
+                throw new Exception($"Failed to revode  provider {ukprn} permission for account legal entity {accountLegalEntityPublicHashedId} ");
```

Comment:
I understand you like to express in emoticons, but please do not expect people to read your mind. Be explicit in your comments so it can be addressed, else I am happy to just ignore.

Reply:
revoke sp.?

Reply:
I figured that much :-) but I wasn't sure if that is the only bit he was concerned about.

Reply:
Yes the spelling of `revoke`